sycl/test/extensions/group-algorithm.cpp

```diff
@@ -76,13 +76,13 @@ int main() {
   std::iota(input.begin(), input.end(), 0);
   std::fill(output.begin(), output.end(), 0);
 
-  test<class KernelNamePlusV>(q, input, output, plus<>(), 0, GeZero());
-  test<class KernelNameMinimumV>(q, input, output, minimum<>(),
+  test<class KernelNamePlusV>(q, input, output, ONEAPI::plus<>(), 0, GeZero());
```

Comment:
For the sake of consistency the functors used in this file should be changed to `sycl::` too
```suggestion
  test<class KernelNamePlusV>(q, input, output, sycl::plus<>(), 0, GeZero());
```

Reply:
There ONEAPI functions are used (like reduce, etc). Will these changes satisfy consistency if functions will remain the same and will use sycl:: function objects?

Reply:
No, now I that think of it, it makes sense to keep these as is, since we separate group algorithm and group algorithm 2020 tests.